### PR TITLE
Add list of demo materials for easing presentation of KubeVirt at events

### PR DESCRIPTION
**Pull request by iranzo**
Add index of materials, presentations so that future presenters have a cheatsheet for quick access to resources that can be used in a demonstration of KubeVirt

### CODE REVIEW

File: docs/demomaterials/README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  - [Console access or UI](#console-access-or-ui)
+  - [Development](#development)
+
+<!-- /TOC -->
+
+## Purpose of this repo
+
+This repository and its contents, aim to provide instructions and materials to be able to host a demo on KubeVirt technology.
+
+It contains links to different materials already available on the website or videos in the KubeVirt channel that could help set up a demonstration environment
```
Comment (petrkotas) [inline docs/demomaterials/README.md]: Should the sentence end with punctuation?

File: docs/demomaterials/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+**Note**: Videos from YouTube can be downloaded 'locally' for offline usage via `youtube-dl` or similar tools so that's also an option for booth display.
```
Comment (petrkotas) [inline docs/demomaterials/README.md]: s/so that's/so, that's

File: docs/demomaterials/README.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+## Demo
+
+In this section you'll find link to resources to prepare your demonstration about KubeVirt technology
```
Comment (petrkotas) [inline docs/demomaterials/README.md]: s/section you/section, you/

Comment (petrkotas) [inline docs/demomaterials/README.md]: s/link/links/

File: docs/demomaterials/README.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+### Offline resources
+
+#### Technology hands-on
+
+For offline demonstrations you can use:
+
+- [MiniKube](https://kubevirt.io/quickstart_minikube)
+- [Kind](https://kubevirt.io/quickstart_kind)
+
+Both of them makes it easy to have attendees to deploy KubeVirt using virtual machine or containers
```
Comment (petrkotas) [inline docs/demomaterials/README.md]: s/containers/containers./

File: docs/demomaterials/README.md (hunk 1 above)
Comment (iranzo) [reply docs/demomaterials/README.md]: done in next push

File: docs/demomaterials/README.md (hunk 2 above)
Comment (iranzo) [reply docs/demomaterials/README.md]: done in next push

File: docs/demomaterials/README.md (hunk 4 above)
Comment (iranzo) [reply docs/demomaterials/README.md]: done in next push

File: docs/demomaterials/README.md (hunk 3 above)
Comment (iranzo) [reply docs/demomaterials/README.md]: done in next push

**Comment by iranzo** [comment]
Thanks @petrkotas new commit uploaded with your comments addressed!

**Comment by iranzo** [comment]
@ptrnull I've added links with the description instead of the full link, is this what you suggested?

**Comment by ptrnull** [comment]
@iranzo what I meant is to add some more text next explaining what the user may find when visiting that link, for example:
[Live Migration](https://kubevirt.io/user-guide/#/installation/live-migration?id=live-migration): In the user-guide page you may find how to enable Live Migration, it's limitations and some use cases like checking the status of the migration or canceling the migration.

File: docs/demomaterials/README.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
+_Table of contents_
```
Comment (dankenigsberg) [inline docs/demomaterials/README.md]: do you have a link to this new page? If not, please add one.

Comment (iranzo) [reply docs/demomaterials/README.md]: @dankenigsberg which page? to the table of contents?

**Comment by dankenigsberg** [comment]
Yes. I think that this new page is better linked from somewhere else in this repo

**Comment by iranzo** [comment]
> Yes. I think that this new page is better linked from somewhere else in this repo

Not yet, I'll add to material-overview in repository-root and upload new version, thanks for the suggestion!

**Pull request merged**